xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -370,9 +372,16 @@ The default proxy is used only when <xref:System.Net.Http.SocketsHttpHandler.Use
         <ReturnType>System.TimeSpan</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the keep alive ping delay. The client will send a keep alive ping to the server if it doesn't receive any frames on a connection for this period of time. This property is used together with <see cref="System.Net.Http.SocketsHttpHandler.KeepAlivePingTimeout"/> to close broken connections.</summary>
```

Comment:
I am not sure, but maybe it will sound a bit more clear if we say `This property is used together with <see cref="System.Net.Http.SocketsHttpHandler.KeepAlivePingTimeout"/> to check whether the connection is broken.`?